{"name": "test_check_login", "status": "failed", "statusDetails": {"message": "AssertionError: Response time 846.641ms exceeded 200ms\nassert 846.641 < 200", "trace": "api_data = {'accessToken': 'eyJhbGciOiJSU0ExXzUiLCJjdHkiOiJKV1QiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiemlwIjoiREVGIn0.GWfLdDY69UIV-qE8y...X6_UprABzpmjxsCyaTA', 'baseUrl': 'https://api-prod.hezzl.com/', 'campaignId': '145602', 'email': 'test@hezzl.com', ...}\ncheck_response_time = <function check_response_time.<locals>._check_response_time at 0x000001C43646D800>\n\n    @allure.description(\"Метод CheckLogin\")\n    def test_check_login(api_data, check_response_time):\n        with allure.step(\"Отправка запроса на проверку логина\"):\n            response = requests.post(\"https://api-prod.hezzl.com/auth/v1/game/145602/check-login\", json={\"login\": api_data[\"email\"], \"type\": \"email\"})\n        with allure.step(\"Проверка статуса кода\"):\n            assert response.status_code == 200, \"Expected status code 200\"\n        with allure.step(\"Получение accessToken из ответа\"):\n            api_data[\"accessToken\"] = response.json().get(\"accessToken\")\n        with allure.step(\"Проверка наличия accessToken\"):\n            assert api_data[\"accessToken\"]\n        with allure.step(\"Проверка времени ответа\"):\n>           check_response_time(response)\n\ntests\\test_hezzl.py:71: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, max_time = 200\n\n    def _check_response_time(response, max_time=200):\n        response_time = response.elapsed.total_seconds() * 1000\n>       assert response_time < max_time, f\"Response time {response_time}ms exceeded {max_time}ms\"\nE       AssertionError: Response time 846.641ms exceeded 200ms\nE       assert 846.641 < 200\n\nconftest.py:20: AssertionError"}, "description": "Метод CheckLogin", "steps": [{"name": "Отправка запроса на проверку логина", "status": "passed", "start": 1709280416543, "stop": 1709280417391}, {"name": "Проверка статуса кода", "status": "passed", "start": 1709280417391, "stop": 1709280417391}, {"name": "Получение accessToken из ответа", "status": "passed", "start": 1709280417391, "stop": 1709280417391}, {"name": "Проверка наличия accessToken", "status": "passed", "start": 1709280417391, "stop": 1709280417391}, {"name": "Проверка времени ответа", "status": "failed", "statusDetails": {"message": "AssertionError: Response time 846.641ms exceeded 200ms\nassert 846.641 < 200\n", "trace": "  File \"C:\\Users\\checa\\PycharmProjects\\api\\tests\\test_hezzl.py\", line 71, in test_check_login\n    check_response_time(response)\n  File \"C:\\Users\\checa\\PycharmProjects\\api\\conftest.py\", line 20, in _check_response_time\n    assert response_time < max_time, f\"Response time {response_time}ms exceeded {max_time}ms\"\n"}, "start": 1709280417391, "stop": 1709280417391}], "start": 1709280416543, "stop": 1709280417392, "uuid": "db8e0101-04c1-4689-86e7-7339dfe75327", "historyId": "4f38890596c81e78b70d7c6008f48629", "testCaseId": "5aa94687bda4438a3e8b29dbb64b5b98", "fullName": "tests.test_hezzl#test_check_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_hezzl"}, {"name": "host", "value": "checamaro"}, {"name": "thread", "value": "5160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_hezzl"}]}